plugins/woocommerce-blocks/bin/blocks-json-to-php.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
I'm worried about breaking changes when updating to `30.3.0`. I'm not 100% sure it's an issue because the way the monorepo splits up packages and plugins, but checking the [changelog](https://github.com/WordPress/gutenberg/blob/HEAD/packages/scripts/CHANGELOG.md), there are a few breaking changes. I think it's okay because `./woocommerce-blocks` uses React 18 (2022-12-14 breaking changes) and doesn't use `lint-md-js` (2022-05-04), do you know how to test if it's safe to upgrade?

Reply:
I found something else that makes me not want to upgrade: The `28.0.0 (2024-05-31)` breaking change says:
```
Note If you're using @wordpress/scripts for building JS scripts to target WordPress 6.5 or earlier, you should not upgrade to this version and continue using @wordpress/scripts@27.
```
And WC still supports 6.5 per [this document](https://woocommerce.com/document/update-php-wordpress/) and slack convo. p1730824429807539-slack-C03CPM3UXDJ

Reply:
Ah, I see anything merged to trunk now won't be released until WC 9.5, which will support WP 6.6. p1730828228999269/1730824429.807539-slack-C03CPM3UXDJ

Went ahead and updated it to use my code from wp-scripts.